src/ray/raylet/local_task_manager.cc

```diff
@@ -228,11 +237,6 @@ void LocalTaskManager::DispatchScheduledTasksToWorkers() {
               internal::UnscheduledWorkCause::WAITING_FOR_RESOURCES_AVAILABLE);
           break;
         }
-        num_unschedulable_task_spilled_++;
```

Comment:
Why do we move this code?

Reply:
This is moved into `TrySpillback()` since this needs to be called whenever spill happens. By moving it inside TrySpillback() we make sure it always happens when spill also we avoid the code duplication since TrySpillback() now is called in two places.